Index.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="setting.css">
+    <title>landing-page</title>
+</head>
+<body>
+    <header class="header">
+        <img src="./product-landing-page-logo.png" alt="" id="header-img">
+        <nav class="nav-bar">
+<ul>
+    <li><a  href="https://product-landing-page.freecodecamp.rocks/#features">Features</li></a>
+    <li><a  href="https://product-landing-page.freecodecamp.rocks/#features">How It Works</li></a>
+    <li><a  href="https://product-landing-page.freecodecamp.rocks/#features">Pricing</li></a>
+</ul>
+     </nav>
+    </header>
+
+    <section>
+    <div class="form">
+        <h2>Handcrafted, home made Masterpieces</h2>
+        <form id="form" action="https://www.freecodecamp.com/email-submit">
+            <input
+            type="email"
+            placeholder="Enter your email addres"
+            name="email"
+            id="email"
+            readonly
+            />
+            <input id="submit" type="submit" value="Get Started" />          
+        </form>
+    </div> 
+
+</section>
+    	
+    
+    <div class="icon">
+        
+     <div>
+        <div class="Premium">
+            <div >
+                <img src="icon1.JPG" alt="" id="header-img">
+            </div>
+            <div>
+                <ul>
+               <h4 class="title"><a href="">Premium Materials</a></h4>
+               <p > Our trombones use the shiniest brass which is sourced locally. This will increase the longevity of your purchase</p>  
+               </ul>
```

Comment:
> your work is good can you change instead
> 
> tag use the
> tag command b/c no list order

- [ ] You should make sure your comments are clear and easy to understand, so the owner of the code understand them clearly. :100: 
**Example of good comment :+1:**
```bash
Your work look good ,it could be better to modify and align the margin, the padding property of the layout so it looks smart. You could also align all the input field in the centre :+1:
```